code-for-canada/project-thundercat
# [Look and Feel] CAT Home Page

# Description

Updating the CAT home page to match the look and feel of https://trello.com/c/AnV6sJuG/28-build-cat-home-page

## Type of change

Please delete options that are not relevant.

- Code cleanliness or refactor

## Screenshot

Before:
![image](https://user-images.githubusercontent.com/2746350/53092240-3c463c80-34e2-11e9-89f0-3b78a462687d.png)


After:
![image](https://user-images.githubusercontent.com/2746350/53090593-ec657680-34dd-11e9-9f16-f1e7284c99e6.png)


# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Go to Home page.
2.  Verify that the home page looks correct

Screenshot of unit tests run passing:
![image](https://user-images.githubusercontent.com/2746350/53091546-351e2f00-34e0-11e9-919f-c48de9ac4276.png)

# Checklist

Applicable for all code changes.
